### ISSUE TITLE
Feature request: Blender Addon

### ISSUE THREAD

**Issue by remissao** [airbnb/lottie-web#1833]
I found a Python framework that includes tools to edit and create Lottie animations.
This framework includes a Blender addon (see here: https://gitlab.com/mattia.basaglia/tgs/#blender) that allows exporting Lottie animations from Blender.
At this point, the addon is still quite limited, but it shows tremendous potential.
Better support for this addon would open new possibilities for creating impressive animations for Lottie as well as giving Lottie access to whole new community of artists.

**Comment by nerddiana** [comment]
Yes, please, add support for Blender and make Lottie more accesible for all, not all people can use After Effects :C

**Comment by AB3-Dev** [comment]
Yes! I would love something like this! I fully support this!
